modular_skyrat/modules/opposing_force/code/opposing_force_datum.dm

```diff
@@ -0,0 +1,845 @@
+		return
+	if(!ispath(processed_item, /obj/item))
+		send_system_message("Error: [processed_item] is not an item")
+		return
+
+	send_system_message("Here are the item specifications for [type]:")
+	send_system_message("Name: [initial(processed_item.name)]")
+	send_system_message("Description: [initial(processed_item.desc)]")
+	send_system_message("Weight class: [initial(processed_item.w_class)]")
+	send_system_message("Tool behaviour: [initial(processed_item.tool_behaviour)]")
+	send_system_message("Weak against armor: [initial(processed_item.weak_against_armour) ? "Yes" : "No"]")
+	send_system_message("Damage type: [initial(processed_item.damtype)]")
+	send_system_message("Wound bonus: [initial(processed_item.wound_bonus)]")
+	send_system_message("Bare wound bonus: [initial(processed_item.bare_wound_bonus)]")
+	send_system_message("Force: [initial(processed_item.force)]")
+
+/datum/opposing_force/proc/unlock_equipment(mob/user)
+	if(!check_rights_for(user.client, R_ADMIN))
+		send_system_message("ERROR: You do not have permission to do that.")
+		return
+	if(!equipment_issued)
+		send_system_message("ERROR: Equipment not yet issued.")
+		return
+	equipment_issued = TRUE
+	send_system_message("Equipment unlocked.")
+
+/datum/opposing_force/proc/ping_admin(mob/user)
+	if(!handling_admin)
+		send_system_message("ERROR: No admin is handling the application.")
+		return
+	if(!COOLDOWN_FINISHED(src, ping_cooldown))
+		send_system_message("ERROR: Ping is on cooldown.")
+		return
+	if(request_updates_muted)
+		send_system_message("ERROR: You are muted.")
+		return
+	if(user.ckey != handling_admin && GLOB.directory[handling_admin])
+		to_chat(GLOB.directory[handling_admin], span_pink("OPFOR: [user] has pinged their OPFOR admin chat! (<a href='?src=[REF(src)];admin_pref=show_panel'>Show Panel</a>)"))
+		SEND_SOUND(GLOB.directory[handling_admin], sound('sound/misc/bloop.ogg'))
+		send_system_message("Handling admin pinged.")
+		COOLDOWN_START(src, ping_cooldown, OPFOR_PING_COOLDOWN)
+	else
+		send_system_message("ERROR: Ping failed.")
+
+/datum/opposing_force/proc/ping_user(mob/user)
+	if(!check_rights_for(user.client, R_ADMIN))
+		send_system_message("ERROR: You do not have permission to do that.")
+		return
+	send_system_message("User pinged.")
+	to_chat(mind_reference.current, span_pink("OPFOR: [get_admin_ckey(user)] has pinged your OPFOR chat, check it!"))
+	SEND_SOUND(mind_reference.current, sound('sound/misc/bloop.ogg'))
+
+/datum/opposing_force/proc/roundend_report()
+	var/list/report = list("<br>")
+	report += span_greentext(mind_reference.current.name)
+	if(objectives.len)
+		report += "<b>Had an approved OPFOR appliation with the following objectives:</b><br>"
+		for(var/datum/opposing_force_objective/opfor_objective in objectives)
+			if(opfor_objective.status != OPFOR_OBJECTIVE_STATUS_APPROVED)
+				continue
+			report += "</b>Title:<b> [opfor_objective.title]<br>"
+			report += "</b>Description:<b> [opfor_objective.description]"
```

Comment:
the \<b>'s are backwards, causing the entirety of OPFOR endrounds to be bolded.
Plus, honestly it just doesn't look very readable regardless. It ends up a big wall of text, with Title Description over and over; perhaps the title could be bolded, then the description below it not bold?

That way it'd result in something like

**Do the thing**
The thing makes me money
**Win**
good

Reply:
if you wanna take a shot at making it better, that'd be cool

Reply:
I'd love to buT THIS ISNT MERGED YET REEEE
and making a branch off anything but master scares me